cg/constants/lims.py

```diff
@@ -2,6 +2,8 @@
 
 from cg.utils.enums import StrEnum
 
+YES_NO_LIMS_BOOLEANS = ["require_qcok", "tumour", "verified_organism"]
```

Comment:
```suggestion
YES_NO_LIMS_BOOLEANS = ["require_qc_ok", "tumour", "verified_organism"]
```

Reply:
Sounds, good! Asked @moedarrah to fix the front-end part 👌

Reply:
Fixed in the front-end 👍